src/core_plugins/kibana/public/management/sections/indices/_edit.js

```diff
@@ -67,7 +64,7 @@ uiModules.get('apps/management')
   });
 
   $scope.$watchCollection('indexPattern.fields', function () {
-    $scope.indexPattern.fields.filter(type => { type: 'conflict'; });
+    $scope.indexPattern.fields.filter(() => { type: 'conflict'; });
```

Comment:
Introduced in https://github.com/elastic/kibana/pull/10746/files#diff-9b4fde59e99415bf74d81aa6ba803797

/cc @thomasneirynck @w33ble